lib/libvfio-user.c

```diff
@@ -1209,6 +1220,29 @@ exec_command(vfu_ctx_t *vfu_ctx, vfu_msg_t *msg)
     return ret;
 }
 
+static int
+do_reply(vfu_ctx_t *vfu_ctx, vfu_msg_t *msg, int reply_ret)
```

Comment:
this arg should just the errno (or 0) too

Reply:
ok